packages/ui-rating/src/RatingIcon/index.tsx

```diff
@@ -41,6 +41,7 @@ import { allowedProps, propTypes } from './props'
 parent: Rating
 id: Rating.Icon
 ---
+@tsProps
```

Comment:
I remember from last year that you had this 2 branches mixed up a bit, but now accidentally you have a change for RatingIcon in the spinner PR :) Can you please revert this file on this branch?